src/adapters/scene.js

```diff
@@ -87,6 +95,14 @@ Scene.prototype.initMapBox = function () {
 
     this.mb.on('moveend', () => {
       UrlState.replaceUrl()
+      fire(
+        'store_center',
```

Comment:
```suggestion
       fire('store_center', {
          lng: this.mb.getCenter().lng,
          lat: this.mb.getCenter().lat,
          zoom: this.mb.getZoom()
        })
```

I prefer compact notation